### ISSUE TITLE
Incorrect link on hackage

### ISSUE THREAD

**Issue by qrilka** [phaazon/smoothie#12]
For some reason link from "Contents" e.g. at http://hackage.haskell.org/package/smoothie-0.4.2.2/docs/Data-Spline-Curve.html goes to docs/index.html and not to the main package version page like it does for other package on hackage

**Comment by phaazon** [comment]
Yeah, I noticed that. The fix would be to regenerate the documentations directly on hackage, and I have no idea how to do that. :(
